src/index.tsx

```diff
@@ -198,7 +198,7 @@ function Dashboard() {
jH5gvKGVBvhHcuXXcrn5FXz2vIlV9MYIqk6GlmSc3PCGxVVO7w3bWRm+SJu5HXofPkBU8h8AVbqFwG45gswKJFarQFSWL7sctYgr+5iyxcq9JalvyI9O5cuZZ3Zf62r5pr9lvcvg858YTfysnxxZfnyVXGi8aCmiHxB+ExKXVaQxvJlpx/k18jyJTdzmVuWdO1c6h9BtHBBKjDYv6czX9iNvCxfXFm+fN2ZmFdkjW9pF1H5AkzUzNb+1zvCNpYvu1yRV8GJ5Us+Hr1Pm78o2ijYWnwreenlj66Cjhxz+DIozRduIy/LF1eWLzmI6yIeonvLuE4fE5cvmF2SOs+nSGf54mDYI69uIsb58ijmssRgI+y65DQrMZMtzk/N1pHNpAwGtOYLs5GX5Ysjy5d8hDX+v4/VENCXL0BX2QmY0gu2s3zZbUJ+1RjnS20m5CzzuaA3vH/kWMRki/dTc+E78pnJ7Xtq82XYJK8eI8uXz7B8yckD89/HuxNAZ76gsubfju5qFWxl+eLknvzqMM4X4ELAbe/9utALx94LhfRiigWTp+aCd2SLzYuP1OYLr5GX5Ysry5fcHDFeBShNALX5Apx9IyU+HjpbvjgJS+RVKeScL/z3ugY1+ect/nPBfLK1xVjsbcnpwrnIKVdz5P/Ivv58wYL86li+fILlS36iEdM3wTfmFajOF6Cr4jULpQk+ZvnipEN+3bPOF+57XZ8eoEz3kaQpHatZ//rnWt45yaA2Ax+K84XTyMvyxZXlS57C7wwvUW7MZ4D2fEE7YdqOXOdsmvMFVfJrwjtfOO91vRF+T2+qcp3x0nyKXsJn5J+nkysSZXEGThTnC7rk172/fElgLF9+mXGba/6OlwLkC1CZirlVicOcTXW+RDfkVXPIPF+ADsv37ZbqWg6LvxEJ+nUK1qo2773SEbQQdsft2jfN+cJo5GX54sjyJXfXt4wC5ne8FCRfgHDDcPHLTXN+CgeWL44uyK+rmHu+cAyYfqLgsuRtZhsZARNUr6FYPBESMHf8zh6pzpeoRV71hpYvGVm+7MFwzeR/qrmZAQXKF2C2Fhkw/VEIJ5Yvrr6TXwn/fOEWMM1pBaqFU/4bXBvrU2j3IuDuvasB+FGdL3xGXpYvjixf9iIcMbjbpzQNgYLlC5OPPpvWMoIjyxdX7TF5FTwIyBegw+YMTH+kd8fSPxXmv07NDaeT4rli3O3vfeO2bawI+YIN+ZVYvmRl+bIX0fKGvLo6KgMFzBcGH30240kZrixf3K0a5NW4LSFfuARMa6nzrPg7UcL3EH9/WoCC/GvA+BD/gusrW5Xni++RV2Nl+ZKJ5cvexJ1FQJ701mcACpovQPwi5j0wTx1kY/niKiG/5jLyBVjVmuTX+ZG+W3q3Kr+wCMZ3zo8VHzxKsaqy3Gjcm/M9eqQ8X/AQkFfjtuVLNpYv+3M6LZEH58cRUOR8ATAcCViC6ddOkJHli7P4ifwaCMkXIKqPyZvgnuM2/716YPfkHNzz3K60V5WkRcyM65xXIbXnC5ORl+WLK8uXvYq7zwEdVL96AqDw+QJgcMvtGeGVYNFtIzPLF3fXTfKqVZGSLwAuPH1bWqNinLZ4I2T15Nyf6j+vnyruclqnZx/y6vOFx8jr0PkygrF8SRcun+hQ+tVOGbB8+Suqsz2hebcM8RmWLxkck1+XgvIFCJMbOrBg0SnQrrGf7d2H1ppAEAVgQYgBCyolIAIqID9FVBTm/d8s9eSkx0KV+z2CHti9MzvLz5a62Y7yipgZ/To19rNN2pIgGTmtz5AvH1/aUfJCfLkZ4kv1to5E1WOm1ue9AOLLj8ZFCydl1UkLPqbcg/gySKhZxy7Fl09Gkx3Vxw9av1+r1pxvfn5caryO0jxNUKhh67yt4/r9ii+DBTXLQ3y5A+JLPcZOLFKFmFxbtuvhbEd8GQyGet6S8tpXa68dJwT6EF9Oa2rU6n234ssnb2lIdQjd9k4o3+QVZvSYNtRR2mDOciI1Rt4bw0EX9CC+dKXkBVCrGeutqQqrLHgbwD/Doy1SC4ix24UiG/TaNY2oWuHkOoDvgTGiJuyKHk7r/92MT2RqgGKyL/61VgB4AUsttWUq0xrR5Tbvjh5DjWK8I9Yp6ISxwylUDZm7oO/yi00QU70kF32X38yP5opqxQh6n+eOAKBThpqTrKkMTBagiHmPUSpRM0TbwV8FXbIcOYlCJVOnepsvhm3QjM9WVA8luWwH8EdDKw+pJlKh9fbiCgDoqjGf2yt6mOILgYWRywecjImtUK0UuzCwZ4MO+hxhVlSSaMr2fFT//782J1O1xNjVujFn0Zz3B0+liu2EI9ZvAOiqreHuI4XuIkema6B29pTlmBckkWog+/kBhzSgy5ZvhzPH0FPW9kTHwclbfLDSWKZqiH6B3teNNos9QxVRTR4rOAB032zEOoIdMSL9naj62dQ96FdUbMoy15y9StURI2ExQp0TXsPMugjxiu4mhqajo+lyl+HbwouoXDsz0DBlcZ/tsSi7U7/iUqOXn2kFgFf2bqPph0vguGmRC9Nz6gYXnjV0a3TFC68iJ2tRZKFMpVKk/YTXUOaEl7O1Dk6+91WR/kuJsvNFH2PL/KC55ZiRTM8Tw72ro+z1oOWVn8ZrKgFj5yzurAAAgNIst9blnOxEetJKMtODhqwJL2/2pvPuJBc8M+FiP9oxa0YNIz+2E+/sLAxtjP1yGZYbw/F8hR4jS6Z7HKMD/LyZtsgfP0CpJmdew9lJAACoxHJjsUEqZHG4otuJTGSbubs4atixAUD5TiMjOO99hm7D+FkeHLX3uNaqXPOxzqeevZPpJkrICS5vbdCABACAegxPV4sN3Ml5KnhmltixH4Uqo+4iybe5ZG96Ql6kDm9oGxTVAKAWw/djzTgE6dRMODv2pXD3+aUUSn5sc4k5TQPe0K7vsV2u2mwz0tmLe/a+LA1StFMZRg2/LQ5C4SxYa4SVAR70EcC8CJSGenDjAAAAAElFTkSuQmCC"
```

Comment:
Why not put it to `public/` folder?

Reply:
I don't think it necessary.

Reply:
hmmm it's not so friendly to the code editor(

Reply:
I also wanna remind here not use base64 image in injected components. Cause image will not reveal to user before stylesheets adopted.

Reply:
nothing will base on this commit and even itself will be dropped within this week... So I don't think any further processing is necessary.